src/components/SetStateDialog.vue

```diff
@@ -206,4 +205,19 @@ export default {
   color: #000;
   font-weight: bold !important;
 }
+
+.checkbox-container {
+  margin-left: 30px;
+  margin-top: -20px;
+}
+/* stylelint-disable */
+.v-btn__loader {
```

Comment:
Editing this vuetify class risks affecting every button loader in the app (same for the theme light subheader above.  You can see the affect of this in the flow tile if you delete a label from this branch - the loader is not visible (but is in dev).  Elsewhere we get around this issue by placing the Vuetify class inside of a parent class - Happy to pair on this if you're not sure what I mean putting it in inside a parent class (or there's examples in other places e.g. the flow summary tile.)